### PR TITLE
FNBC-36: Sound Effects & Soundtrack

### PR DESCRIPTION
**Pull request by sahilsharma09**

### CODE REVIEW

File: Client.py [hunk 1]
```diff
@@ -25,6 +26,20 @@ def __init__(self):
...
         self.suggested = False
+        pygame.init()
+        pygame.mixer.music.load("sounds/Clue-Less_Soundtrack.mp3")
+        pygame.mixer.music.set_volume(0.03)
+        pygame.mixer.music.play(-1)
+        self.suggest_sound = pygame.mixer.Sound('sounds/Suggest.wav')
+        self.accuse_sound = pygame.mixer.Sound('sounds/Accuse.wav')
+        self.move_sound = pygame.mixer.Sound('sounds/Move.wav')
+        self.won_sound = pygame.mixer.Sound('sounds/Won.wav')
+        self.lost_sound = pygame.mixer.Sound('sounds/Lost.wav')
```
Comment (bwiswell) [inline Client.py:37]: Using the "/" character to reference subdirectories will cause errors on some systems/installations. Consider using a solution such as:
import os
sound_path = os.path.dirname(os.path.realpath(__file__)) + "\\sounds\\"
pygame.mixer.music.load(sound_path + "Clue-Less_Soundtrack.mp3")

File: Client.py [hunk 2]
```diff
@@ -25,6 +26,20 @@ def __init__(self):
...
+        pygame.mixer.Sound.set_volume(self.suggest_sound, 0.01)
+        pygame.mixer.Sound.set_volume(self.accuse_sound, 0.01)
+        pygame.mixer.Sound.set_volume(self.move_sound, 0.01)
+        pygame.mixer.Sound.set_volume(self.won_sound, 0.01)
+        pygame.mixer.Sound.set_volume(self.lost_sound, 0.01)
```
Comment (bwiswell) [inline Client.py:42]: Consider replacing the magic number 0.01 with a client attribute (i.e. self.sound_volume = 0.01) to make future volume adjustments easier.

File: Client.py [hunk 3]
```diff
@@ -25,6 +26,20 @@ def __init__(self):
         self.lost = False
         self.myNumber = None
         self.suggested = False
+        pygame.init()
+        pygame.mixer.music.load("sounds/Clue-Less_Soundtrack.mp3")
+        pygame.mixer.music.set_volume(0.03)
```
Comment (bwiswell) [inline Client.py:31]: Consider replacing the 0.03 with a client attribute (i.e. self.music_volume = 0.01) to make future volume adjustments easier.

**Review by bwiswell — CHANGES_REQUESTED** [review]

File: Client.py (hunk 1 above)
Comment (bwiswell) [reply Client.py:37]: Interestingly, GitHub has changed this comment slightly. The "\" characters around 'sounds' should be double backslashes. The bolded 'file' should be preceded and followed by two underscores. See Card.py for an example of this syntax.

Comment (sahilsharma09) [reply Client.py]: Completed

File: Client.py (hunk 2 above)
Comment (sahilsharma09) [reply Client.py]: Completed

File: Client.py (hunk 3 above)
Comment (sahilsharma09) [reply Client.py]: Completed

**Review by bwiswell — APPROVED** [review]

**Pull request merged**